src/planscape/plan/views.py

```diff
@@ -425,6 +434,73 @@ def get_project_areas(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
 
 
+def _save_scenario_metadata(max_budget, max_treatment_area_ratio, max_road_distance, max_slope, priorities, weights, scenario: Scenario):
```

Comment:
nit: the auto-formatter should be able to split this into multiple lines.

Reply:
1: weird! it doesn't catch it, but fixed manually
2: done thanks!

Reply:
thanks!
I don't see this in the code yet but I assume it's a matter of pushing the local repo.